### ISSUE TITLE
Drop EOL python django support

### ISSUE THREAD

**Issue by Mogost** [aldjemy/aldjemy-wip#4]
django-python versions compatibility [matrix](https://docs.djangoproject.com/en/3.0/faq/install/#what-python-version-can-i-use-with-django):

Django version | Python versions
-- | --
1.11 | 2.7, 3.4, 3.5, 3.6, 3.7 (added in 1.11.17)
2.0 | 3.4, 3.5, 3.6, 3.7
2.1 | 3.5, 3.6, 3.7
2.2 | 3.5, 3.6, 3.7, 3.8 (added in 2.2.8)
3.0 | 3.6, 3.7, 3.8

[Python EOL](https://python-release-cycle.glitch.me/)
![image](https://user-images.githubusercontent.com/721963/72269646-00a1ed00-3635-11ea-907b-8689ff121a18.png)

[Django EOL](https://www.djangoproject.com/download/#supported-versions)
![image](https://user-images.githubusercontent.com/721963/72269660-0566a100-3635-11ea-8d24-c6f9c2b0c529.png)

**Comment by ryanhiebert** [comment]
I agree with your desire here, but I'd like us to wait on implementing this for a little bit, see if we can do some non-breaking cleanup changes first, to give some time to see if we come up to a resolution on whether aldjemy will be moved here, or if its on board with the idea in general. That way, if he has some input on what versions he wants to see supported, and we can take that into account.
